src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -481,6 +483,39 @@ definitions:
     dependencies:
       accessKeyId: [ secretAccessKey ]
       secretAccessKey: [ accessKeyId ]
+  Camunda:
+    type: object
+    properties:
+      pr:
+        "$ref": "#/definitions/CamundaCluster"
+      test:
+        "$ref": "#/definitions/CamundaCluster"
+      acceptance:
+        "$ref": "#/definitions/CamundaCluster"
+      production:
+        "$ref": "#/definitions/CamundaCluster"
+      diagramResourcesPath:
```

Comment:
If you put these at the same level as the clusters, I don't think it's very clear what 'pr' 'test' etc stand for?

Reply:
I have update the structure with the consideration that the next step is to set a post-deployment step using another sets of credentials and might even need to include another sets of paths